test/assert.js

```diff
@@ -182,6 +182,110 @@ test('.throws()', function (t) {
 	t.end();
 });
 
+test('.throws() returning Error', function (t) {
+	t.doesNotThrow(function () {
+		var assertion = assert.throws(function () {
+			throw new Error('foo');
+		});
+
+		assert.same('foo', assertion.message);
```

Comment:
Yea. No need to pass any secondary arguments either, the existing tests don't seem to bother with that either.